core/src/replay_stage.rs

```diff
@@ -1727,20 +1727,27 @@ impl ReplayStage {
         // All errors must lead to marking the slot as dead, otherwise,
         // the `check_slot_agrees_with_cluster()` called by `replay_active_banks()`
         // will break!
-        blockstore_processor::confirm_slot(
-            blockstore,
-            bank,
-            &mut w_replay_stats,
-            &mut w_replay_progress,
-            false,
-            transaction_status_sender,
-            Some(replay_vote_sender),
-            None,
-            verify_recyclers,
-            false,
-            log_messages_bytes_limit,
-            prioritization_fee_cache,
-        )?;
+
+        let mut more_entries_to_process = true;
```

Comment:
could do bank.is_complete here, but should've been picked up after-the-fact on the previous iteration and not marked for replay